### ISSUE TITLE
kodi - No match found on URL: https://kodi.tv/download/

### ISSUE THREAD

**Issue by blackthroat** [autopkg/andrewvalentine-recipes#7]
No match found on URL: https://kodi.tv/download/

WARNING: MakeCatalogs.munki is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...

**Comment by blackthroat** [comment]
Changing Line 13 in kodi.download.recipe 

from
`<string>https://kodi.tv/download/</string>`
to
`<string>https://kodi.tv/download/851/</string>`

I believe fixes it.

**Comment by andrewvalentine** [comment]
Thanks for raising this and providing the solution. This has been fixed in commit `3a87f9c`.

This message:
```
WARNING: MakeCatalogs.munki is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
```

relates to:

https://github.com/autopkg/autopkg/wiki/AutoPkg-and-recipe-parent-trust-info

> AutoPkg 1.x expects you to verify and trust the recipes you run. This is new behavior compared to the 0.x releases.
Make overrides for every recipe you run. Run autopkg update-trust-info override_name to add trust info to any existing override.

This should be fixed in your local overrides. The Autopkg wiki has all the information you need on this.
